doc/config.example.yaml

```diff
@@ -35,25 +37,25 @@ node:
   stats_listening_port: 9110
   # The new block time offset has to be greater than the previous block time offset
   # but less than current time + block_time_offset_tolerance_secs
-  block_time_offset_tolerance_secs: 60
+  block_time_offset_tolerance_seconds: 60
```

Comment:
In my opinion using the new approach for all `Durations` is better as it not only makes it consistent, the names are shorter, and it removes the need to debate if a new config field should be limited to a certain unit.

Reply:
I'll add a commit to switch them all. But I think in some situations, it might make sense to restrict things to a certain unit. Also, it wouldn't surprise me if some people had the exact opposite PoV. Since I want to isolate the config framework in its own repository, and the work is already there, perhaps let's keep it but not use it ?

Reply:
Yes I was going to suggest that perhaps by convention we do not use in our `Agora` to keep it consistent, but I agree it might be useful for other projects to have this option.